syntax = "proto3";
package xrpc.example;

service UserService {
    rpc Login(LoginRequest) returns (LoginResponse);
}

message LoginRequest {
    string username = 1;
    string password = 2;
}

message LoginResponse {
    bool result = 1;
    string message = 2;
}